cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_initial.spec.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+const workspaceName = 'test_workspace';
+let workspaceId;
+let workspaceFeatures = ['use-case-observability'];
+
+if (Cypress.env('WORKSPACE_ENABLED')) {
+  describe('Workspace initial', () => {
+    before(() => {
+      cy.deleteWorkspaceByName(workspaceName);
+      cy.createWorkspace({
+        name: workspaceName,
+        features: workspaceFeatures,
+        settings: {
+          permissions: {
+            library_write: { users: ['%me%'] },
+            write: { users: ['%me%'] },
+          },
+        },
+      }).then((value) => (workspaceId = value));
+    });
+
+    beforeEach(() => {
+      // Visit workspace initial page
+      miscUtils.visitPage(`/app/home`);
+    });
+
+    after(() => {
+      cy.deleteWorkspaceById(workspaceId);
+    });
+
+    it('should contain correct content', () => {
+      // contains initial page title and description
+      cy.contains('Welcome to OpenSearch').should('exist');
+      cy.contains('My workspaces').should('exist');
+      cy.contains(
+        'Collaborate on use-case based projects with workspaces. Select a workspace to get started.'
+      ).should('exist');
+
+      // contains five use case title
+      cy.contains('Observability').should('exist');
+      cy.contains('Security Analytics').should('exist');
+      cy.contains('Search').should('exist');
+      cy.contains('Essentials').should('exist');
+      cy.contains('Analytics').should('exist');
+
+      // contains no workspace message
+      cy.contains('No workspaces').should('exist');
```

Comment:
Nit: Do we need to check which card have no workspaces message? Since we created a observability workspace before. This use case should not show "No workspaces".

Reply:
If we can get the workspace name, the no workspace message of the use case must not exist. So the check for the observability is not necessary.